prow/config.yaml

```diff
@@ -5761,6 +5761,47 @@ presubmits:
           requests:
             memory: "2Gi"
 
+  - name: pull-test-infra-verify-config-canary
```

Comment:
Seems like this can also be added in a separate PR.

Reply:
If we care, sure, I just had it here to test the generation

Reply:
Also shows us literally what the config looks like